optashift-employee-rostering-webapp/pom.xml

```diff
@@ -61,11 +61,6 @@
       <artifactId>optashift-employee-rostering-restclient</artifactId>
       <scope>test</scope>
     </dependency>
-    <dependency>
```

Comment:
I am pretty sure webapp has Unit tests

Reply:
I move the junit dependency to the top level pom.
Junit (and maybe mockito) is the only "default" dependency (but only in test scope) for all modules.